build.zig

```diff
@@ -300,7 +300,7 @@ fn configureStage2(b: *Builder, exe: var, ctx: Context) !void {
     }
     dependOnLib(b, exe, ctx.llvm);
 
-    if (exe.target.getOs() == .linux) {
+    if (exe.target.getOs() == .linux or exe.target.getOs() == .dragonfly) {
```

Comment:
The line below "On Fedora, install libstdc++-static and try again." should only ever appear if the target is Linux.

Reply:
Fixed!